tests/frontend/test_kibana_app/cypress/integration/utils/driver.js

```diff
@@ -106,4 +111,14 @@ export const getMyCookie = () => {
     return cookie = c;
   })
 }
-
+// Function that's return the selector by xpath
+export const xPathSelector = (selector) => {
```

Comment:
could this function be called something like
"findElementByXpath"
since it better describes the action it is performing

Reply:
Changes: 
xpathSelector -> findElementByXpath